### ISSUE TITLE
The react-pdf-html doesn't see registered font. 

### ISSUE THREAD

**Issue by AntonBorisovArcuman** [danomatic/react-pdf-html#27]
I'm getting this error
`Font family not registered: 'Open Sans'. Please register it calling Font.register() method`
 when trying to run this code 
```
const html = `<html>
  <style>
    body {
      font-family: 'Open Sans';
    }
  </style>

  <body>
    <p>
        Paragraph with <strong>bold</strong>, <i>italic</i>, <u>underline</u>,
        <s>strikethrough</s>,
    </p>
  </body>
</html>`
```
```
return (
    <Document onRender={renderCallBack}>
      <Page>
        <Html>{html}</Html>
      </Page>
      </Document>
```

```
const customFonts = [
  {
    family: 'Open Sans',
    src: '/fonts/OpenSans-Light.ttf',
  },
  {
    family: 'Open Sans',
    fontWeight: 'bold',
    src: '/fonts/OpenSans-SemiBold.ttf',
  },
  {
    family: 'Open Sans',
    fontStyle: 'italic',
    src: '/fonts/OpenSans-SemiBold.ttf',
  },
  {
    family: 'Open Sans',
    fontStyle: 'italic',
    fontWeight: 'bold',
    src: '/fonts/OpenSans-SemiBold.ttf',
  },
]

customFonts.forEach((font) => {
  Font.register(font)
})
```
If I use this font without lib, then this font is applied and display correctly. Also if I removed font from body, then `<b> and <i> `displaying incorrectly. I'm using @react-pdf/renderer
![image](https://user-images.githubusercontent.com/88824760/153263914-fbfb39fe-ced7-49a1-9405-b3fa66feeab2.png)

**Comment by pekez** [comment]
Same here

**Comment by danomatic** [comment]
Any style settings should pass through, so this would need further digging to reproduce and resolve. Assistance is welcome as I'm very busy ATM.

**Comment by Tomcat88** [comment]
@danomatic Any idea on where to start looking?

**Comment by quangsiDev** [comment]
same here

**Comment by danomatic** [comment]
the issue is the quoted string. it's coming into the components as a style of `{ fontFamily: "'Open Sans'" }` with double quoting. Temporary fix is to use `OpenSans` with no quotes but I'm working on a real fix now.

**Comment by danomatic** [comment]
Fixed in version `1.1.9`
